Example/Views/Pages/UserDetails.xaml

```diff
@@ -1,34 +1,73 @@
-                <StackLayout BackgroundColor="{DynamicResource Primary}" VerticalOptions="End" HorizontalOptions="FillAndExpand">
-                    <Label x:Name="username" Text="" FontAttributes="Bold" FontSize="14" TextColor="{DynamicResource InversePrimaryTextColor}" XAlign="Center" />
-                    <Label x:Name="email" Text="" FontSize="10" TextColor="{DynamicResource InversePrimaryTextColor}" XAlign="Center" />
-                </StackLayout>
-                <StackLayout 
-                    VerticalOptions="FillAndExpand"
-                    BackgroundColor="White">
-                    <Label Text="User Roles" FontSize="10" TextColor="Gray" BackgroundColor="White" FontAttributes="Bold" />
-                    <ListView x:Name="lstRoles" BackgroundColor="White" />
-                    <Button Text="Logout" 
-                        BackgroundColor="{DynamicResource Primary}"
-                        TextColor="{DynamicResource InversePrimaryTextColor}"
-                        Clicked="onLogoutClicked"/>
+             xmlns:controls="clr-namespace:TwinTechsForms.NControl;assembly=TwinTechsForms.NControl.SvgImageView" 
+             xmlns:utils="clr-namespace:Example.Utils" 
+             Title="Identity Profile" BackgroundColor="#3b96b1" x:Name="UserDetailsPage">
+    
+    <StackLayout VerticalOptions="FillAndExpand" Padding="0,0,0,0" Spacing="0">
+        <StackLayout Orientation="Horizontal" HorizontalOptions="FillAndExpand" BackgroundColor="White" Margin="10,10,10,0" Padding="0,0,0,0" Spacing="0">
+            <utils:TintedSvgImage TintColor="{DynamicResource Primary}" WidthRequest="30" HeightRequest="30" Source="resource://Example.Resources.person.svg" Margin="5,10,5,0" />
+
+            <!--Full Name-->
+            <StackLayout Spacing="0" HorizontalOptions="FillAndExpand" BackgroundColor="White">
+                <Label Text="Full Name" TextColor="{DynamicResource Primary}" FontSize="Micro" Margin="0,10,0,0" />
+                <StackLayout Orientation="Horizontal">
+                    <Label Text="{Binding CurrentUser.Firstname}" FontSize="Small" />
+                    <Label Text="{Binding CurrentUser.Lastname}" FontSize="Small" />
                 </StackLayout>
             </StackLayout>
-            <circleimg:CircleImage x:Name="Image" Source="aerogear_icon.png" Aspect="AspectFill" WidthRequest="80" HeightRequest="80" HorizontalOptions="Center" RelativeLayout.XConstraint="{ConstraintExpression Type=RelativeToParent, Property=Width, Factor=0.5, Constant=-40}" RelativeLayout.YConstraint="{ConstraintExpression Type=RelativeToParent, Property=Y, Factor=0, Constant=16}" />
-        </RelativeLayout>
-    </ContentPage.Content>
+        </StackLayout>
+        <StackLayout Spacing="0" Margin="10,0,10,0" BackgroundColor="White">
+
+            <!--Email-->
+            <Label BackgroundColor="White" TextColor="{DynamicResource Primary}" FontSize="Micro" Text="Email" Margin="10,10,10,0" />
```

Comment:
We can now use also StringResources from XAML. It's dirty solution and works only for Text attribute, but it's based on the Xamarin documentation (simplified by fact that there is no translation to different languages).

xmlns:s="clr-namespace:Example.Utils;assembly=Example"
```xml
<Label Text="{s:StringResource UnderConstructionTitle}" 
```